plugins/modules/iam_role.py

```diff
@@ -17,14 +17,25 @@
 options:
   path:
     description:
-      - The path to the role. For more information about paths, see U(https://docs.aws.amazon.com/IAM/latest/UserGuide/reference_identifiers.html).
-    default: "/"
+      - The path of the role.
+      - For more information about IAM paths, see the AWS IAM identifiers documentation
+        U(https://docs.aws.amazon.com/IAM/latest/UserGuide/reference_identifiers.html).
+      - Updating the path on an existing role is not currently supported and will result in a
+        warning.
+      - C(path_prefix) and C(prefix) were added as aliases in release 7.2.0.
     type: str
+    aliases: ["prefix", "path_prefix"]
   name:
     description:
-      - The name of the role to create.
+      - The name of the role.
+      - >-
+        Note: Role names are unique within an account.  Paths (I(path)) do B(not) affect
```

Comment:
```suggestion
        Note: Role names are unique within an account.  Paths I(path) do B(not) affect
```
are these supposed to be wrapped in ()? I haven't seen it before, so I could be wrong though with suggestion

Reply:
I wrapped them in brackets to avoid it getting rendered as "Paths path do not affect..", instead it'll render as "Paths (path) do not affect..."